llvm-core/clang/clang-15.0.7-r3.ebuild

```diff
@@ -69,11 +69,6 @@ llvm.org_set_globals
 # Therefore: use llvm-core/clang[${MULTILIB_USEDEP}] only if you need
 # multilib clang* libraries (not runtime, not wrappers).
 
-pkg_setup() {
-	LLVM_MAX_SLOT=${LLVM_MAJOR} llvm_pkg_setup
```

Comment:
I'm pretty sure `llvm` inherit can be removed entirely now (which solves another problem — the use of deprecated eclass).